src/test/java/de/tum/in/www1/artemis/exercise/AthenaResourceIntegrationTest.java

```diff
@@ -286,7 +304,7 @@ void testRepositoryExportEndpoint(String urlSuffix) throws Exception {
         // Get exports from endpoint
         var authHeaders = new HttpHeaders();
         authHeaders.add("Authorization", athenaSecret);
-        var repoZip = request.getFile("/api/public/athena/programming-exercises/" + programmingExercise.getId() + "/" + urlSuffix, HttpStatus.OK, new LinkedMultiValueMap<>(),
+        var repoZip = request.getFile(ATHENA_PROGRAMMING_EXERCISE_REPOSITORY_API_PATH + programmingExercise.getId() + "/" + urlSuffix, HttpStatus.OK, new LinkedMultiValueMap<>(),
```

Comment:
Is that intended? This seams to be the other way round than the PR wants to go

Reply:
No, Thanks. I overuse Strg + R here.